doc/api/errors.md

```diff
@@ -2706,6 +2706,183 @@ size.
 This `Error` is thrown when a read is attempted on a TTY `WriteStream`,
 such as `process.stdout.on('data')`.
 
+<a id="openssl-error-codes"></a>
+## OpenSSL Error Codes
+
+<a id="Time Validity Errors"></a>
+### Time Validity Errors
+
+<a id="CERT_NOT_YET_VALID"></a>
+#### `CERT_NOT_YET_VALID`
+
+The certificate is not yet valid: the notBefore date is after the current time.
+
+<a id="CERT_HAS_EXPIRED"></a>
+#### `CERT_HAS_EXPIRED`
+
+The certificate has expired: that is the notAfter date is before the current
+time.
+
+<a id="CRL_NOT_YET_VALID"></a>
+#### `CRL_NOT_YET_VALID`
+
+The revocation data have a future issue date.
```

Comment:
```suggestion
The revocation data has a future issue date.
```

Reply:
Technically "have" is more corerct. I'll reword it to avoid the awkwardness of "data".